bundles/paikkatietoikkuna/dimension-change/request/DimensionChangeRequest.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @class Oskari.mapframework.bundle.dimension-change.DimensionChangeRequest
+ */
+Oskari.clazz.define('Oskari.mapframework.bundle.dimension-change.DimensionChangeRequest',
```

Comment:
We don't need this anymore, right?

Reply:
True that, removed the file and it's entry from the bundle package.